src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
-body {
+$main-color: #060b35;
+$secondary-color: #616070;
+$white-color: #fff;
+$lightblue-color: #00acdc;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 32px 16px 16px;
+  color: $main-color;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__product-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    color: $secondary-color;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    color: $main-color;
+    padding: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    color: $white-color;
+    text-transform: uppercase;
+    background-color: $lightblue-color;
+    padding: 12px 70px 12px 69px;
+    text-decoration: none;
+    border: none;
+    font-weight: 700;
+    font-size: 14px;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    line-height: 16px;
```

Comment:
let's improve your code. Please check this link https://codeguide.academy/html-css.html#css-order.
It's about CSS properties order.